set.mm

```diff
@@ -19216,6 +19215,29 @@ Converse of the inference rule of (universal) generalization ~ ax-gen .
       ( wal spimfv alrimiv ) ACGBDABCDEFHI $.
   $}
 
+  ${
```

Comment:
This points out that substitution has two implementation variants.  I wonder whether it is best placed before df-sb then.

I'd prefer equs5v, or maybe equsb45v, over subst as a name